egov-apportion-service/src/main/resources/application.properties

```diff
@@ -48,3 +50,6 @@ egov.mdms.search.endpoint=/egov-mdms-service/v1/_search
 
 #Default apportion switch
 egov.apportion.default.value.order=true
+
+management.endpoints.web.base-path=/
+spring.main.allow-bean-definition-overriding=true
```

Comment:
This cannot be set to `true`, if there is an error then you need to make sure only bean is loaded